### PR TITLE
[RA]: Fix Reference Architecture link in start page

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ Anuket Specifications
     Technical Specifications </common/index>
     Introduction </common/chapter00>
     Reference Model </ref_model/index>
-    Reference Architectures </ref_arch/index>
+    Reference Architectures </ref_arch/README>
```
[the remark this answers is not in the document]
Comment (karinesevilla) [reply doc/index.rst:10]: +1

**Comment by CsatariGergely** [comment]
> Sorry but I don't understand your point here. If we consider sphinx, it should be index.rst. There is a falsy link to README to please github user (ideally we should have renamed README.rst index.rst). but all-in-all we will fully README like the bundly history in GSMA...

@collivier at the moment there is no index.rst. I will create a symlink to README.md and use that.

**Review by pgoyal01 — APPROVED** [review]

**Review by ildikov — APPROVED** [review]

**Review by collivier — CHANGES_REQUESTED** [review]
@CsatariGergely I don't get your point. The issue is more on the caller side.
We don't need a specific index.rst as there is no specific sphinx project. We could rename all README index but it will diverge from default Github rendering 

We could gate ra top dir as rc top dir but I would ask for tox.ini test-requirements, etc.

**Comment by collivier** [comment]
@CsatariGergely yes the issue is more on https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/README.rst

**Review by rgstori — APPROVED** [review]

**Comment by CsatariGergely** [comment]
Sphinx requires an index.rst while GitHub assumes a README.md . I copied this solution with linking from RI or RC (I do not remember anymore) in the hope that it will work for RA-s too. 
Can you @collivier explain what solution you expect to get this merged? At the moment RA2 is not accessible in readthedocs due to this bug, so this would need a quicker fix. 
Thanks.

**Comment by CsatariGergely** [comment]
@collivier can you please check my comment. We really need to have a resolution for this. 
@scottsteinbrueck please review. 

Thanks.

**Comment by gkunz** [comment]
@CsatariGergely shouldn't it be the other way around: have all the real content in index.rst and then a link to a README to allow rendering on GitHub. This would also be consistent with RC1/2 and (somewhat) with RI1/2.

**Comment by CsatariGergely** [comment]
> @CsatariGergely shouldn't it be the other way around: have all the real content in index.rst and then a link to a README to allow rendering on GitHub. This would also be consistent with RC1/2 and (somewhat) with RI1/2.

Okey, let's try that. Done in [000672e](https://github.com/cntt-n/CNTT/pull/2962/commits/000672ef73e60278820bb9cc28baa54592d195a6)

**Review by walterkozlowski — APPROVED** [review]

**Comment by collivier** [comment]
> Can you @collivier explain what solution you expect to get this merged? At the moment RA2 is not accessible in readthedocs due to this bug, so this would need a quicker fix.

> @collivier can you please check my comment. We really need to have a resolution for this.

The previous solution implemented in RCX and in RA1 seems good. If you consider sphinx as first choice, README.rst should be a link to index.rst. In RA1 it was the opposite firstly focusing priorly on backward compatibility.

About the overall thread: the main problem here is why the main doc isn't gated as such and why we allow the doc to be merged and published with incorrect labels or document unused.

You can spam me to merge but I would never agree to merge a an RST file named MD. I could have allowed on the suffix .txt  - Github render index.rst too...

**Comment by walterkozlowski** [comment]
@collivier, can you please check and approve if it is ok for merge, thanks

**Review by collivier — APPROVED** [review]

**Pull request merged**